Dockerfile

```diff
@@ -2,4 +2,6 @@ FROM python:2.7-slim
 COPY . /app
 WORKDIR /app
 RUN pip install --trusted-host pypi.python.org -r requirements.txt
-RUN apt-get update; apt-get install ssh-client wget -y
+RUN apt-get update; apt-get install ssh-client wget gpg -y --no-install-recommends
```

Comment:
Worthwhile cleaning up the apt cache at end of this RUN command `&& rm -rf /var/lib/apt/lists/*`.
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

Reply:
Good catch